### PR TITLE
Adds more adminbus smites to keep the admins happy

### PR DESCRIPTION
**Pull request by mystery3525**
## About The Pull Request

Adds some new smites:
- Adminheal
- Admindamage
- Animalize
- Give Cookie
- Give Stalker Trauma
- Force Sleep
- Force Pacifism
- Forcesay
- Floorclwune variant
- Voice of God

A good portion of these already exists, but being able to Right Click > Smite to quickly do something is really cool and easy

If we come up with any more ~~crapola~~ great ideas we'll add it.

## Why It's Good For The Game

May or may not be good for the game, but definitely great for admins

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

</details>

## Changelog
:cl:
add: Several new admin smites
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
Do not let this feature shove them onto glass tables

**Comment by Tyranicranger4** [comment]
> Do not let this feature shove them onto glass tables

Literally 1984.

**Comment by EvilDragonfiend** [comment]
> > Do not let this feature shove them onto glass tables
> 
> Literally 1984.

unless an admin wants all random glass tables in the station broken lol

**Comment by Rukofamicom** [comment]
Get a proper chrono legionaire smite up and running, remove players from the round in style.

**Comment by mystery3525** [comment]
> Get a proper chrono legionaire smite up and running, remove low-risk but offending players from the round in style.

what do you mean 😳

**Comment by Rukofamicom** [comment]
It spawns someone outfitted in the Chrono Legionnaire equipment to hunt down the smited person. After hunting the person down, the Legionnaire vanishes themselves. Their equipment doesn't work on anyone except the hunted target and the player is removed "From the timeline entirely" which is to say the same as if they had cryo'd will occur to objectives and manifest.

It is usually used it to remove people with bad character names, such as Hitler or Minecraft Steve ("This isn't your universe" or "Your bloodline is a threat to the universe") but can be used to somewhat immersively remove any banned player in a way that's more interesting for the round, so long as the banned player isn't at risk of griefing in response to the ban. 

image

**Comment by mystery3525** [comment]
yeah we decided against that because 
1. legionnare is broken (or so I have been told)
2. Why would a griefer or someone named "BAN EVADER" ever actually roleplay meaningfully with someone meant to round remove them
3. creating a custom antagonist designed to hunt down a shitter is outside the scope of this pr

**Comment by Rukofamicom** [comment]
> Why would a griefer or someone named "BAN EVADER" ever actually roleplay meaningfully with someone meant to round remove them

I can understand deciding against it, but this is downright insulting/bad faith.

**Comment by itsmeow** [comment]
>     1. legionnare is broken (or so I have been told)

https://github.com/BeeStation/BeeStation-Hornet/pull/7673

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -64,7 +64,7 @@
 		to_chat(usr, "The person you are trying to contact is not wearing a headset.")
 		return
 
-	if (!sender)
```
Comment (itsmeow) [inline code/modules/admin/verbs/randomverbs.dm]: i hate this so much

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -1126,92 +1280,49 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
+	else
+		qdel(cookie)
+		log_admin("[key_name(src)] has their hands full, so they did not receive their cookie, spawned by [key_name(admin_client)].")
+		message_admins("[key_name_admin(src)] has their hands full, so they did not receive their cookie, spawned by [key_name(admin_client)].")
+
 /client/proc/punish_log(var/whom, var/punishment)
 	var/msg = "[key_name_admin(usr)] punished [key_name_admin(whom)] with [punishment]."
-	message_admins(msg)
 	admin_ticket_log(whom, msg)
+	message_admins(msg)
```
Comment (itsmeow) [inline code/modules/admin/verbs/randomverbs.dm]: message_admins should use ADMIN_LOOKUPFLW instead of key_name_admin

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by FortSumter** [comment]
"I understand the council has made a decision, but considering it's a stupid decision I've elected to ignore it"

Chrono legionnaire isn't meant to be entertaining for the offender, but for everyone else. It is a sad day when you can't even make banning someone fun.

**Comment by itsmeow** [comment]
It's unrelated to the PR, it's not a "no" it's a "this is not part of the scope of the PR"

File: code/modules/admin/verbs/randomverbs.dm [hunk 3]
```diff
@@ -64,7 +64,7 @@
 		to_chat(usr, "The person you are trying to contact is not wearing a headset.")
 		return
 
-	if (!sender)
+	if(!sender)
```
Comment (PowerfulBacon) [inline code/modules/admin/verbs/randomverbs.dm]: Spaces between if statements should not be changed for no reason. See code standards for more details.

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by mystery3525** [comment]
@itsmeow reopen please I have been resurrected and addressed the reviews in latest push

File: code/modules/admin/verbs/randomverbs.dm [hunk 4]
```diff
@@ -136,7 +136,7 @@
 
 	var/msg = capped_input(usr, "Message:", "Enter the text you wish to appear to everyone:")
 
-	if (!msg)
+	if( !msg )
```
Comment (itsmeow) [inline code/modules/admin/verbs/randomverbs.dm]: wtf is this

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/randomverbs.dm (hunk 4 above)
Comment (mystery3525) [reply code/modules/admin/verbs/randomverbs.dm]: github fuckery

**Review by itsmeow — DISMISSED** [review]

**Comment by itsmeow** [comment]
```
code/modules/admin/verbs/randomverbs.dm, line 1307, column 19:
error: undefined var: "whom"

code/modules/admin/verbs/randomverbs.dm, line 1307, column 73:
error: undefined var: "whom"

code/modules/admin/verbs/randomverbs.dm, line 1308, column 70:
error: undefined var: "whom"

code/modules/admin/verbs/randomverbs.dm, line 1309, column 48:
error: undefined var: "whom"
```

**Review by itsmeow — APPROVED** [review]

**Pull request merged**